_includes/header.html

```diff
@@ -1,217 +1,181 @@
-                  />
-                </svg>
-              </a>
-            </div>
-            <div class="syn-dropdown-list-item syn-body-2 syn-mb-0">
-              <a href="https://learn.algorithmia.com" target="_blank"
-                >Learning Center
-                <svg class="syn-ml-4" width="14" height="14" viewBox="0 2 24 20">
-                  <path
-                    fill="currentColor"
-                    d="M19 19H5V5h7V3H5c-1.11 0-2 .9-2 2v14c0 1.1.89 2 2 2h14c1.1 0 2-.9 2-2v-7h-2v7zM14 3v2h3.59l-9.83 9.83 1.41 1.41L19 6.41V10h2V3h-7z"
-                  />
-                </svg>
-              </a>
-            </div>
-          </div>
+      </div>
+    </div>
+  </div>
+  <div onclick="['nav-open'].forEach((a) => document.querySelector('.syn-page-header').classList.toggle(a))"></div>
+  <div class="syn-page-header-content syn-flex justify-space-between">
+    <div class="syn-flex align-center">
+      <a class="syn-flex" href="https://algorithmia.com">
+        <img class="syn-ml-8" src="https://algorithmia.com/v1/config/logo" style="max-height: '24px'; max-width: '100%';" />
+      </a>
+    </div>
+    <button class="syn-btn icon text grow syn-hidden-lg" onclick="['nav-open'].forEach((a) => document.querySelector('.syn-page-header').classList.toggle(a))">
+      <svg width="32" height="20" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" class="open-button"><g stroke="currentColor" stroke-width="1.5" fill="none" fill-rule="evenodd" stroke-linecap="round"><path d="M1 1h29.849M1 10h29.849M1 19h29.849"></path></g></svg>
+      <svg width="24" height="24" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" class="close-button"><g stroke="currentColor" stroke-width="1.5" fill="none" fill-rule="evenodd" stroke-linecap="round"><path d="M1.371 22.553L22.478 1.447M1.371 1.461l21.107 21.107"></path></g></svg>
+    </button>
+    <div class="syn-visible-lg">
+      <a href="https://algorithmia.com/mlops" class="syn-link-no-decoration">
+        <button class="syn-btn text">MLOps</button>
+      </a> 
+      <a href="https://algorithmia.com/product" class="syn-link-no-decoration">
+        <button class="syn-btn text syn-ml-8">Product</button>
+      </a>
+      <a href="https://algorithmia.com/pricing" class="syn-link-no-decoration">
+        <button class="syn-btn text syn-ml-8">Pricing</button>
+      </a>
+      <div class="syn-dropdown syn-ml-8">
+        <button class="syn-btn text" onclick="this.parentNode.classList.toggle('open')">
+          Learn
+          <svg width="7px" height="4px" viewBox="0 0 7 4" class="syn-ml-4">
+            <polygon fill="currentColor" points="0 0 6.688 0 3.344 3.344"></polygon>
+          </svg>
+        </button>
+        <div class="open-break syn-bg-black syn-mt-8"></div>
+        <div class="syn-dropdown-list" onclick="this.parentNode.classList.toggle('open')">
+          <div class="syn-dropdown-list-item syn-body-2 syn-mb-0">
```

Comment:
```suggestion
          <li class="syn-dropdown-list-item syn-body-2 syn-mb-0">
```

Reply:
Not going to add suggested changes to each list item, but these should all be `li` elements.